src/static/js/app/api/SurveyApi.js

```diff
@@ -0,0 +1,26 @@
+import 'fetch';
+
+export default class SurveyApi {
+  static getFirstPageForUserId(userId, callback) {
```

Comment:
Well, there you go. Sometimes an annoying snag forces you to an unexpected
insight.

On Sun, Oct 11, 2015 at 10:12 PM, Graham Holtslander <
notifications@github.com> wrote:

> In src/static/js/app/api/SurveyApi.js
> <https://github.com/holtscomm/survey/pull/20#discussion_r41721804>:
>
> > @@ -0,0 +1,26 @@
> > +import 'fetch';
> > +
> > +export default class SurveyApi {
> > +  static getFirstPageForUserId(userId, callback) {
>
> Making these static so they can be called like
> SurveyApi.getFirstPageForUserId and such.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/holtscomm/survey/pull/20/files#r41721804>.
>



-- 
--
Gord
Gordon Holtslander, BEd
Partner, Holtslander Communications
holtslandercommunications.ca
gordon@holtslander.com